devmaster/modules/devshop/devshop_projects/drush/devshop_provision.drush.inc

```diff
@@ -136,27 +135,29 @@ function devshop_provision_provision_services() {
  * @param $data
  *   Associative array of data from Provision_Config_Apache_Site::data.
  *
- * @return
+ * @return string|null
  *   Lines to add to the configuration file.
  *
  * @see Provision_Config_Apache_Site
  * provision_logs_provision_apache_vhost_config
  */
 function devshop_provision_provision_apache_vhost_config($uri, $data) {
 
-  $environment = d()->environment;
-  $project = d()->project;
+  $context = d();
+  $environment = $context->environment;
+  $project = $context->project;
 
   if (empty($environment) || empty($project)) {
-    return;
+    return NULL;
```

Comment:
Does `return;` and `return null;` have the same effect?

Reply:
Yes, they do. I wrote `NULL` to appease PhpStorm's warning.